nwb_conversion_tools/baserecordingextractorinterface.py

```diff
@@ -31,7 +31,8 @@ def get_metadata_schema(self):
         metadata_schema = super().get_metadata_schema()
 
         # Initiate Ecephys metadata
-        metadata_schema['properties']['Ecephys'] = dict(
+        metadata_schema['properties']['Ecephys'] = get_base_schema(tag='Ecephys')
```

Comment:
why is this tagged?

Reply:
Not sure, doesn't look familiar - this looks like a change inherited from elsewhere, I'll try to track it down.